express-api/src/services/parcels/parcelServices.ts

```diff
@@ -90,10 +90,8 @@ const updateParcel = async (incomingParcel: DeepPartial<Parcel>) => {
   if (findParcel == null || findParcel.Id !== incomingParcel.Id) {
     throw new ErrorWithCode('Parcel not found', 404);
   }
-  await parcelRepo.update({ Id: findParcel.Id }, incomingParcel);
-  // update function doesn't return data on the row changed. Have to get the changed row again
-  const newParcel = await getParcelById(incomingParcel.Id);
-  return newParcel;
+
+  return parcelRepo.save(incomingParcel);
```

Comment:
```suggestion
  return await parcelRepo.save(incomingParcel);
```
This feels safer if we await the promise.

Reply:
The caller will need to await the returned Promise of this function either way.